### ISSUE TITLE
[BUG] Language of bible books only used in bible text window, not in mag view, not in search

### ISSUE THREAD

**Issue by tuxuser06** [bibletime/bibletime#349]
Language of bible books only used in bible text window, not in mag view, not in search

**Steps to reproduce**
Configure BT to use german names of the bible books ("Sprache der Bibelbücher-Namen: Deutsch").
Use bible GerHfa2002.
Go to math 1:1. It has a link to math 20,30-31.
Hover over the link with the mouse.
The link appears in the mag view.
In the mag view its titled english: "Matthew 20:30-Matthew 20:31", it should be german: "Matthäus..."
Same problem in search:
Open search, use bible GerHfa2002.
Search for "Matthäus".
In the results you see "Matthäus...",.. "Lukas 6:15". That's expected, it's german.
Click "Lukas 6:15".
You see the bible text. Here the bible name is english, it's "Luke" but should be "Lukas".

System is openSuse 15.3
Bibletime 3.0.2
Sword version is 1.8.1
Qt Version is 5.12.7

**Comment by jaakristioja** [comment]
I can't reproduce this on my system. Does this also happen with other modules? We don't have access to the encrypted GerHfa2002. If possible, please post some screenshots as well.

**Comment by tuxuser06** [comment]
I found an other module with links: ISV (english bible (International Standard Version)).
Go to math 1:11. It has a link to 2 kings 24:14-16. 
Hover over the link with the mouse.
The link appears in the mag view.
In the mag view its titled english: "II Kings 24:14-II Kings 24:16;" but it should be german.

**Comment by jaakristioja** [comment]
Hmm... but the module ISV itself is in English, so maybe it is correct to display that in English as well?

**Comment by tuxuser06** [comment]
Sorry, I configured bibletime to use german names of bible books. And the problem also shows up with a german bible. With bibletime 2.0.2 I get german bible book names. As expected.
Another question: how can I append screenshots? This website can't append jpg, it also can't append png. "Something wne really wrong, and we can't process that file:"

**Comment by jaakristioja** [comment]
> Sorry, I configured bibletime to use german names of bible books. And the problem also shows up with a german bible. With bibletime 2.0.2 I get german bible book names. As expected.

I'm using book names in Estonian, and I something similar. In GerAlbrecht, the intros before Matthew 1 have some links which seem to be reference abbreviations in German, and when hovering over them, the Mag displays the same references in English. I suppose this is similar to what you experience?

> Another question: how can I append screenshots? This website can't append jpg, it also can't append png. "Something wne really wrong, and we can't process that file:"

Dunno. Seems to work for me. GitHub's own [guide](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#uploading-assets) seems quite brief. Make sure your browser is not blocking some GitHub domain, e.g. including domains like github-production-user-asset-6210df.s3.amazonaws.com.

**Comment by tuxuser06** [comment]
![bibletime](https://user-images.githubusercontent.com/18015616/148697392-0eb8f0e2-2cd7-43e7-83e2-851ba5ed04a8.png)
Oh, I had a script blocker running. Now image upload seems ok.
What you can see: Every bible book is named german. Except the mag view is english.

**Comment by tuxuser06** [comment]
> I'm using book names in Estonian, and I something similar. In GerAlbrecht, the intros before Matthew 1 have some links which seem to be reference abbreviations in German, and when hovering over them, the Mag displays the same references in English. I suppose this is similar to what you experience?

Yes, that's similar to my problem. It's working ok in the old bibletime 2.0.2.
